testsupport/wait/member.go

```diff
@@ -608,6 +608,19 @@ func UntilNSTemplateSetHasSpaceRoles(expected ...toolchainv1alpha1.NSTemplateSet
 	}
 }
 
+// UntilNSTemplateSetHasNoSpaceRoles returns a `NSTemplateSetWaitCriterion` which checks that the given
+// NSTemlateSet has no space roles.
+func UntilNSTemplateSetHasNoSpaceRoles() NSTemplateSetWaitCriterion {
```

Comment:
this doesn't seem to be used

Reply:
Will delete if not needed but may need it as test evolves as one check is we don't get a parent space role if the sub space has disabled inheritance.

Reply:
Removed it